### PR TITLE
Add example Pipelines config

### PR DESCRIPTION
**Pull request by balsama**

### CODE REVIEW

File: acquia-pipelines.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# This is an example Acquia Pipelines configuration file. For complete
+# documentation and other examples, see https://docs.acquia.com/acquia-cloud/develop/pipelines/yaml/
+#
+# If you'd like a project which sets up testing and pipelines for you, consider
```
Comment (phenaproxima) [inline acquia-pipelines.yml]: Should be "Pipelines".

File: acquia-pipelines.yml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+services:
+  - mysql
+  - php:
+      version: 7.1
+
+events:
+  build:
+    steps:
+    # Build a development codebase suitable for testing.
+    # Uncomment these lines if you have setup dev dependencies and testing for
```
Comment (phenaproxima) [inline acquia-pipelines.yml]: "...if you have to set up dev dependencies..."

File: acquia-pipelines.yml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    # assumes you have a well-configured settings.php file committed to your
+    # repo and all testing dependencies like Behat and PHPUnit as dev
+    # dependencies.
+    #  - test:
+    #      type: script
+    #      script:
+    #        - drush site-install
+    #        - drush runserver --default-server=builtin 8080 &>/dev/null &
+    #        - behat --stop-on-failure
+    #        - phpunit --configuration ./PATH_TO_YOUR_CONFIG --group YOUR_TEST_TAGS
```
Comment (phenaproxima) [inline acquia-pipelines.yml]: This gives the impression like the --group and --configuration options are required for PHPUnit. Is that an impression we want to give? Maybe just call 'phpunit' and assume they have the configuration set up as they want?

File: acquia-pipelines.yml (hunk 2 above)
Comment (balsama) [reply acquia-pipelines.yml]: Reworded this so it's hopefully more clear now.

```
    # Uncomment these lines if you have included testing dependencies in
    # Composer's "require-dev" section.
```

**Pull request merged**